SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1962,6 +1975,47 @@ bool USpatialNetDriver::FindAndDispatchStartupOps(const TArray<Worker_OpList*>&
 		}
 	}
 
+	SelectiveProcessOps(FoundOps);
+
+	if (PackageMap->IsEntityPoolReady() &&
+		GlobalStateManager->IsReadyToCallBeginPlay())
+	{
+		// Return whether or not we are ready to start
+		return true;
+	}
+
+	return false;
+}
+
+bool USpatialNetDriver::FindAndDispatchStartupOpsClient(const TArray<Worker_OpList*>& InOpLists)
+{
+	TArray<Worker_Op*> FoundOps;
+
+	// Search for GSM-related ops we need and process them
```

Comment:
Make this a bit clearer - we are waiting for the entity query for the gsm specifically

Reply:
Addressed.